Assets/Scripts/Managers/PlanetManager.cs

```diff
@@ -92,17 +102,23 @@ public void UpdateHeldAnimals(string animalName)
         //TODO: Use animal name to keep track of what animals are held
         //The held display is EXTREMELY temporary. It is just to show the number and get it functional for now.
         held++;
-        animalDisplay.GetComponent<TMPro.TextMeshPro>().text = held.ToString();
+        Debug.Log(held);
     }
 
     private void DepositHeldAnimals()
     {
         GameSettings.Score += held;
-        held = 0;
-        animalDisplay.GetComponent<TMPro.TextMeshPro>().text = held.ToString();
-        scoreDisplay.GetComponent<TMPro.TextMeshProUGUI>().text =
-            "Temp Score Display: " + GameSettings.Score.ToString();
+        held = 0;//was 0
+        //animalDisplay.GetComponent<TMPro.TextMeshPro>().text = held.ToString();
+        scoreDisplay.GetComponent<TMPro.TextMeshProUGUI>().text = GameSettings.Score.ToString();
         //This is very temporary. Will have a better system in the next increment.
```

Comment:
As this is no longer temporary this can be removed

Reply:
The "This is very temporary" comment? Because it is now gone!